src/fu-util-common.c

```diff
@@ -1402,6 +1402,10 @@ fu_util_device_problem_to_string(FwupdClient *client, FwupdDevice *dev, FwupdDev
 		/* TRANSLATORS: The device cannot be updated due to missing vendor's license." */
 		return g_strdup(_("Device requires a software license to update"));
 	}
+	if (problem == FWUPD_DEVICE_PROBLEM_SYSTEM_INHIBIT) {
+		/* TRANSLATORS: an application is preventing system updates */
+		return g_strdup(_("System is inhibited"));
```

Comment:
I think this string should be expanded.

Reply:
Got any ideas? `All devices are prevented from update by inhibit` perhaps?

Reply:
sounds good